src/coreclr/tests/src/Interop/COM/Dynamic/Server/DispatchImpl.cpp

```diff
@@ -26,7 +26,7 @@ DispatchImpl::DispatchImpl(GUID guid, void *instance, const wchar_t* tlb)
 
 DispatchImpl::~DispatchImpl()
 {
-    if (_typeInfo != nullptr)
+    if (_typeLib != nullptr)
```

Comment:
Good catch.

Reply:
Oh, didn't even notice we actually have our own [`ComSmartPtr`](https://github.com/dotnet/runtime/blob/master/src/coreclr/tests/src/Interop/common/ComHelpers.h#L336).